src/Monitoring/grafana-init/grafana.ini

```diff
@@ -69,3 +69,6 @@ provider = azure_blob
 account_name = dotnetgrafana
 ;account_key = <managed by environment>
 container_name = grafana
+
+[snapshots]
+external_enabled = false
```

Comment:
What's this? Should we re-run the setup to get it pushed to the servers? (I think that will upgrade Kusto, which will cause other problems)

Reply:
On a dashboard, click the "Share" button, the select the "Snapshot" tab. This option turns off the "Publish to snapshot.raintree.io". It's not a big deal, but it seems best to not let our data roam around unless necessary. 

Pushing this out would be good just to have a consistent setup but this is not urgent.